src/Microsoft.Data/GroupBy.cs

```diff
@@ -17,13 +17,13 @@ public abstract class GroupBy
         /// Compute the number of non-null values in each group 
         /// </summary>
         /// <returns></returns>
-        public abstract DataFrame Count();
+        public abstract DataFrame Count(params string[] columnNames);
```

Comment:
These changes seem unrelated. Do we have tests covering this (passing in columnNames)?

Reply:
Yup. The last parts of TestGroupBy